platform/rct/Makefile

```diff
@@ -14,6 +14,13 @@ LD := $(CROSS_COMPILE)g++
 OBJDUMP := $(CROSS_COMPILE)objdump
 OBJCOPY := $(CROSS_COMPILE)objcopy
 
+ASL_FW_POSTPROCESS := asl_f4_fw_postprocess
+ifeq ($(PYTHON),python2.7)
```

Comment:
Ported from the MK2 makefile.  I didn't have it in there originally, so I assumed it was added by Brent/You/Ryan for some compatibility reason.